### PR TITLE
fix-console-errors-due-to-multiple-same-ids

### PR DESCRIPTION
**Pull request by romainlancia**

### CODE REVIEW

File: src/react/databrowser/objects/ObjectDelete.jsx [hunk 1]
```diff
@@ -83,8 +83,8 @@ const ObjectDelete = ({ bucketName, toggled, prefixWithSlash }: Props) => {
                 <Table>
                     <T.Body>
                         {
-                            toggled.map(s => (
-                                <T.Row key={s.name}>
+                            toggled.map((s, index) => (
+                                <T.Row key={`${s.name}-${index}`}>
```
Comment (nicolas2bert) [inline src/react/databrowser/objects/ObjectDelete.jsx]: Why is this needed? `s.key` should be unique.

Comment (romainlancia) [reply src/react/databrowser/objects/ObjectDelete.jsx]: I saw a warning about a non unique key from this line

Comment (nicolas2bert) [reply src/react/databrowser/objects/ObjectDelete.jsx:87]: We should try to avoid `index` when we can. https://reactjs.org/docs/lists-and-keys.html#keys

**Review by nicolas2bert — APPROVED** [review]
Thank you!

**Review by dora-korpar — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

**Comment by romainlancia** [comment]
@bert-e approve
